sdk/python/jobs/finetuning/standalone/model-as-a-platform/chat/chat_completion_with_model_as_platform.ipynb

```diff
@@ -0,0 +1,565 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## FineTuning LLM with Model-As-Service\n",
+    "\n",
+    "This sample shows how use create a standalone FineTuning job to fine tune a model to summarize a dialog between 2 people using samsum dataset.\n",
```

Comment:
What's the meaning of 'standalone' - what are the other options ?

Reply:
Other option is a step inside pipeline, in which case call first goes to FTCloud and then FT service.